### PR TITLE
chore: update dependabot.yml

### PR DESCRIPTION
**Pull request by MSevey**
## Summary by CodeRabbit

- **New Features**
	- Daily updates for `gomod` package dependencies.
	- Grouped patch updates for `npm` and `gomod` into a single pull request.

- **Documentation**
	- Improved clarity and specificity in integration and migration instructions.
	- Updated versioning and import paths in related documents.
	- Enhanced formatting and structure across various documentation files.

- **Bug Fixes**
	- Corrected URLs in migration instructions for the upgrade handler.

- **Style**
	- Minor formatting adjustments in multiple documentation files for consistency.

### CODE REVIEW

**Comment by MSevey** [comment]
Issue for markdown lint link errors https://github.com/rollkit/cosmos-sdk-starter/issues/37

**Review by Manav-Aggarwal — APPROVED** [review]

**Review by tzdybal — APPROVED** [review]

File: sdk/docs/instructions/migration.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ Before starting the migration process, make sure your chain have the following:
 
 - Integrate rollkit-sdk to your chain following [this guideline](./integration.md)
 
-- Integrate `RappDK` upgrade handler to your chain. Example upgrade handler could be found here [upgrade handler](https://github.com/rollkit/cosmos-sdk-starter/sdk
+- Integrate `RappDK` upgrade handler to your chain. Example upgrade handler could be found here [upgrade handler](<https://github.com/rollkit/cosmos-sdk-starter/sdk>
```
Comment (tzdybal) [inline sdk/docs/instructions/migration.md:24]:
```suggestion
- Integrate `RappDK` upgrade handler to your chain. Example upgrade handler could be found here [upgrade handler](<https://github.com/rollkit/cosmos-sdk-starter/tree/main/sdk>
```

File: sdk/docs/instructions/migration.md [hunk 2]
```diff
@@ -21,10 +21,10 @@ Before starting the migration process, make sure your chain have the following:
...
 /blob/01103c74832314f77b3a9271d18c33d393bc0529/simapp/upgrade/upgrade.go#L28).
 
-**Notice: make sure you use correct sequencer pubkey address in your upgrade handler. You need to set the new sequencer pubkey manually in [your upgrade handler logic](https://github.com/rollkit/cosmos-sdk-starter/sdk
+**Notice: make sure you use correct sequencer pubkey address in your upgrade handler. You need to set the new sequencer pubkey manually in [your upgrade handler logic](<https://github.com/rollkit/cosmos-sdk-starter/sdk>
```
Comment (tzdybal) [inline sdk/docs/instructions/migration.md:27]:
```suggestion
**Notice: make sure you use correct sequencer pubkey address in your upgrade handler. You need to set the new sequencer pubkey manually in [your upgrade handler logic](<https://github.com/rollkit/cosmos-sdk-starter/tree/main/sdk>
```

**Review by tzdybal — COMMENTED** [review]
Original SDK links return 404 consistently.

**Review by Manav-Aggarwal — APPROVED** [review]

**Review by Manav-Aggarwal — APPROVED** [review]

**Review by tzdybal — APPROVED** [review]

**Pull request merged**